bower.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "json-refs",
   "main": "browser/json-refs.js",
-  "version": "1.3.0",
```

Comment:
Yeah.  We probably shouldn't mix multiple types of changes in one PR anyways.  If you can remove this part of the PR, I'll merge the rest.